examples/supertuxkart/Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN cd stk-code && mkdir cmake_build && cd cmake_build \
     && make -j$(nproc)
 
 # final image
-FROM debian:buster
+FROM debian:bullseye
```

Comment:
🤔 does it make more sense to do this at the same time we upgrade STK for #2546 ?

The it consolidates the version change and all the Dockerfile changes into a single PR.